### DISCUSSION TITLE
How do I dig into errors reported by Prisma?

### DISCUSSION THREAD

**Discussion by pbelbin** [prisma/prisma#2781] [Q&A] [answered]
Ok, I have some code, and when I run it, I get the following result:

`
2020-06-16T21:59:40.761Z engine stdout { timestamp: 'Jun 16 21:59:40.760',
  level: 'INFO',
  target: 'quaint::connector::metrics',
  fields:
   { query: 'ROLLBACK',
     item_type: 'query',
     params: '[]',
     duration_ms: 2 } }
2020-06-16T21:59:40.762Z engine { error:
   Error: Error occurred during query execution:
   EnvVarNotFound("Expected parent binding \'11\' to be present.")
       at NodeEngine.graphQLToJSError (/app/node_modules/@prisma/client/runtime/index.js:1:93905)
       at currentRequestPromise.then (/app/node_modules/@prisma/client/runtime/index.js:1:92823)
       at process._tickCallback (internal/process/next_tick.js:68:7) }
error! 
Error: Error occurred during query execution:
EnvVarNotFound("Expected parent binding \'11\' to be present.")
    at PrismaClientFetcher.request (/app/node_modules/@prisma/client/runtime/index.js:1:149258)
    at process._tickCallback (internal/process/next_tick.js:68:7)
`

Note, I have the environment variable `DEBUG` set to `*`, and I am seeing a bunch of information prior to the error section I've shown, but, it all appears to be 'successful', from what I can see.  How can I get more *useful* information that will help me know what the real issue is?

None of the information emitted points me to anything in my code at all, from what I can see.

Your help is appreciated :)

**Comment by janpio** [comment, marked as answer]
This might help: https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/logging#overview

In general it is a good idea to open a bug issue and include all the information the template looks for.

**Comment by ludralph** [comment]
Hi @pbelbin,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
